frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -358,10 +358,10 @@ function BookInfo:setCustomBookCover(file, book_props, metadata_updated_caller_c
                 if sidecar_file then
                     sidecar_dir = util.splitFilePathName(sidecar_file)
                 else -- no sdr folder, create new
-                    sidecar_dir = DocSettings:getSidecarDir(file)
+                    sidecar_dir = DocSettings:getSidecarDir(file) .. "/"
                     util.makePath(sidecar_dir)
```

Comment:
Looks a little odd/dirty that we can/do provide a path ending with a / to makePath.
But util.splitFilePathName() already keeps the trailing / on the directory, so I guess everything is just fine with a trailing slash.

Reply:
It has some meaningful semantics for makePath & mkdir, FWIW ;).